docs/api/model_selection/learning_curve.rst

```diff
@@ -110,10 +117,30 @@ Learning curves also work for clustering models and can use metrics that specify
 
 Unfortunately, with random data these curves are highly variable, but serve to point out some clustering-specific items. First, note the y-axis is very narrow, roughly speaking these curves are converged and actually the clustering algorithm is performing very well. Second, for clustering, convergence for data points is not necessarily a bad thing; in fact we want to ensure as more data is added, the training and cross-validation scores do not diverge.
 
+Quick Method
+------------
+The same functionality can be achieved with the associated quick method ``learning_curve``. This method will build the ``LearningCurve`` object with the associated arguments, fit it, then (optionally) immediately show the visualization.
+
+.. plot::
+    :context: close-figs
+    :alt: Learning Curve Quick Method on the Energy dataset using RidgeCV
+
+    from sklearn.linear_model import RidgeCV
+
+    from yellowbrick.datasets import load_energy
+    from yellowbrick.model_selection import learning_curve
+
+    # Load a regression dataset
+    X, y = load_energy()
+
+    # Instantiate the regression model and visualizer
+    model = RidgeCV()
+    visualizer = learning_curve(model, X, y, scoring='r2')
+    visualizer.show()
```

Comment:
`visualizer.show()` is not needed here since it should be called by the quick method.